docs/src/rules/require-unicode-regexp.md

```diff
@@ -98,6 +98,12 @@ function i(flags) {
 
 :::
 
+## Options
+
+This rule has one object option:
+
+* `"requireFlag": "u"|"v"` requires a particular Unicode regex flag
```

Comment:
Can you add a section for this option, with examples of correct and incorrect code, like we have in other rule docs?

Reply:
Sure, added.